src/osemosys.txt

```diff
@@ -56,11 +56,18 @@ set STORAGE;
 param ResultsPath, symbolic default 'results';
 param YearSplit{l in TIMESLICE, y in YEAR};
 
-param DiscountRate{r in REGION, t in TECHNOLOGY};
-param CapitalRecoveryFactor{r in REGION, t in TECHNOLOGY, y in YEAR} :=
-	(1 + DiscountRate[r, t]) ^ (y - min{yy in YEAR} min(yy) + 0.0);
-param CapitalRecoveryFactorMid{r in REGION, t in TECHNOLOGY, y in YEAR} :=
-	(1 + DiscountRate[r, t]) ^ (y - min{yy in YEAR} min(yy) + 0.5);
+param DiscountRate{r in REGION};
+param DiscountRateIdv{r in REGION, t in TECHNOLOGY};
```

Comment:
Note below suggests setting default to that of `DiscountRate` allowing you to potentially remove CC3 and `TypeOfDiscountRate`